api/v1alpha1/clustervirtualmachineimage_types.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (clusterVirtualMachineImage *ClusterVirtualMachineImage) GetConditions() Conditions {
+	return clusterVirtualMachineImage.Status.Conditions
+}
+
+func (clusterVirtualMachineImage *ClusterVirtualMachineImage) SetConditions(conditions Conditions) {
+	clusterVirtualMachineImage.Status.Conditions = conditions
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster,shortName=cvmimage;clustervmimage
+// +kubebuilder:storageversion
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="ContentLibraryName",type="string",JSONPath=".status.ContentLibraryRef.name"
```

Comment:
@cyclexuxu I might have given the wrong example in my previous review. This should be `.status.contentLibraryRef.name` (lowercase) to display that field properly.